content/en/docs/tasks/observability/logs/otel-provider/index.md

```diff
@@ -0,0 +1,197 @@
+---
+title: OpenTelemetry Provider
+description: This task shows you how to configure Envoy proxies to print access logs with OpenTelemetry Provider.
+weight: 10
+keywords: [telemetry,logs]
+aliases:
+    - /docs/tasks/telemetry/access-log
+    - /docs/tasks/telemetry/logs/access-log/
```

Comment:
are these correct?  this is a new doc, right? i think we can skip `aliases` ?

Reply:
yes. fixed